### PR TITLE
Very lightly tested tags, ingredients, and ratings

### PR DESCRIPTION
**Pull request by antTolman**

### CODE REVIEW

**Review by mamberly — APPROVED** [review]
I think this looks great but will this make every recipe have those ingredients? If they put 
"chicken, potatoes, cheese" would that only come up with recipes that have all of those 3 ingredients or would it come up with recipes that have any of those 3? Which one do we want?

File: src/main/kotlin/edu/byu/mealplanningassistant/repository/RecipesRepository.kt [hunk 1]
```diff
@@ -53,16 +54,29 @@ class RecipesRepository{
...
-    private fun getFromDB(filter: Bson? = null) : List<Recipe>{
-        println("filter : $filter")
+    private fun getQueryResults(filter: Bson? = null, sort : Boolean) : Iterator<Document>{
         val collection = getDatabase().getCollection("recipes")
+        return if(sort){
+            (if (filter != null) collection.find(filter).sort(Sorts.descending("rating")).iterator() else collection.find().sort(Sorts.descending("rating")).iterator())
+        } else{
+            (if (filter != null) collection.find(filter).sort(Sorts.descending("rating")).iterator() else collection.find().sort(Sorts.descending("rating")).iterator())
+        }
+    }
```
Comment (Chris4942) [inline src/main/kotlin/edu/byu/mealplanningassistant/repository/RecipesRepository.kt:73]: The two branches of `if(sort)` appear to be exactly the same, is it supposed to be that way?

**Comment by antTolman** [comment]
whoops, looks like I forgot to take the sort call out of one of them.

**Comment by sben502** [comment]
I just pulled this branch to test some of the rating stuff, is the GetRandomizedRecipeRequest in RecipeServiceTest.kt missing the 'ingredients' bit on a few of the lines?

**Pull request merged**